### PR TITLE
Update strings.xml

### PR DESCRIPTION
**Pull request by Kaitul**
Update Traditional Chinese translation

### CODE REVIEW

**Review by bluehomewu — APPROVED** [review]

**Comment by bluehomewu** [comment]
@Mygod Ah....wait , don't merge.
I click wrong.

File: mobile/src/main/res/values-zh-rTW/strings.xml [hunk 1]
```diff
@@ -13,45 +13,49 @@
...
-    <string name="repeater_wps_success_pbc">請在兩分鐘內在需要連接的裝置上，使用一鍵加密以連接到此中繼器</string>
+    <string name="repeater_wps_dialog_pbc">WPS 按鈕</string>
+    <string name="repeater_wps_success_pbc">請在兩分鐘內在需要連線的裝置上，按下 WPS 按鈕以連線到此中繼器</string>
     <string name="repeater_wps_success_keypad">PIN 已設定</string>
-    <string name="repeater_wps_failure">開啟 WPS 失敗 (原因: %s)</string>
-    <string name="repeater_configure_failure">找不到有效的配置，請先啟動中繼器</string>
-    <string name="repeater_clean_pog_failure">刪除多餘 P2P 群組失敗 (原因: %s)</string>
+    <string name="repeater_wps_failure">開啟 WPS 失敗 (原因：%s)</string>
+    <string name="repeater_configure_failure">未找到有效的組態，請先啟動中繼器</string>
+    <string name="repeater_clean_pog_failure">刪除冗餘 P2P 群組失敗 (原因：%s)</string>
```
Comment (bluehomewu) [inline mobile/src/main/res/values-zh-rTW/strings.xml:28]: recommand use `多餘` 
`冗餘` is classical Chinese,

**Review by bluehomewu — APPROVED** [review]

**Comment by Kaitul** [comment]
閣下請看 https://terms.naer.edu.tw/detail/18118342/?index=8 中 redundant 電子計算機之翻譯

File: mobile/src/main/res/values-zh-rTW/strings.xml (hunk 1 above)
Comment (Kaitul) [reply mobile/src/main/res/values-zh-rTW/strings.xml:28]: According to https://terms.naer.edu.tw/detail/18118342/?index=8, “redundant” should be translated into “冗餘”.

**Comment by bluehomewu** [comment]
> 閣下請看 https://terms.naer.edu.tw/detail/18118342/?index=8 中 redundant 電子計算機之翻譯
> 
> According to https://terms.naer.edu.tw/detail/18118342/?index=8, redundant should be translated into “冗餘”.

Hi, I'm Edward.
I think when translate program, we shouldn't use strings that is too classical (formal) for some strings, which is not very friendly to users.

**Comment by Kaitul** [comment]
> > 閣下請看 https://terms.naer.edu.tw/detail/18118342/?index=8 中 redundant 電子計算機之翻譯
> > 
> > According to https://terms.naer.edu.tw/detail/18118342/?index=8, redundant should be translated into “冗餘”.
> 
> Hi, I'm Edward.
> I think when translate program, we shouldn't use strings that is too classical (formal) for some strings, which is not very friendly to users.

But apparently, it's a modern term, and Wikipedia has an entry for it, and it's very common in life: https://www.google.com/search?q=冗餘

**Comment by Mygod** [comment]
I think being formal isn't necessarily a bad thing. Just my two cents.

**Comment by Mygod** [comment]
I assume that there are no further objections? Feel free to submit a new PR otherwise. 🤣

**Pull request merged**